packages/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleValue.tsx

```diff
@@ -65,14 +65,18 @@ function GridFilterInputMultipleValue(props: GridFilterInputMultipleValueProps)
       value={filterValueState}
       onChange={handleChange}
       renderTags={(value, getTagProps) =>
-        value.map((option, index) => (
-          <rootProps.slots.baseChip
-            variant="outlined"
-            size="small"
-            label={option}
-            {...getTagProps({ index })}
-          />
-        ))
+        value.map((option, index) => {
+          const { key, ...tagProps } = getTagProps({ index });
+          return (
+            <rootProps.slots.baseChip
+              key={key}
+              variant="outlined"
+              size="small"
+              label={option}
+              {...tagProps}
+            />
+          );
+        })
```

Comment:
This logic is taken from https://github.com/mui/mui-x/blob/45b28354e18d4178e3ebe2fa33e89fca6e4221dd/packages/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx#L110-L120